backend/app/api/health.py

```
@@ -1,17 +1,22 @@
+from sqlalchemy.ext.asyncio import AsyncSession
 
-from app.core.store import sensor_readings, irrigation_events, sensor_alerts
+from app.core.database import get_db
 
 router = APIRouter(tags=["health"])
 
 
 @router.get("/health")
-async def health_check():
-    """서버 상태 및 인메모리 데이터 현황."""
+async def health_check(db: AsyncSession = Depends(get_db)) -> dict:
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)